src/test/java/net/datafaker/service/FakeValuesServiceTest.java

```diff
@@ -268,7 +268,7 @@ void futureDateExpression() throws ParseException {
         Date now = new Date();
         Date nowPlus10Days = new Date(now.getTime() + MILLIS_IN_A_DAY * 10);
 
-        Date date = dateFormat.parse(fakeValuesService.expression("#{date.future '10','TimeUnit.DAYS'}", faker));
+        Date date = dateFormat.parse(fakeValuesService.expression("#{date.future '10','TimeUnit.DAYS'}",  faker));
```

Comment:
Interesting change 😄

Reply:
This is the actual fix :)

Reply:
nice catch, thanks
I've removed these extra spaces